Console/Command/ResetYotpoSync.php

```diff
@@ -0,0 +1,225 @@
+<?php
+
+namespace Yotpo\Core\Console\Command;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Magento\Framework\App\State as AppState;
+use Yotpo\Core\Model\Config;
+use Yotpo\Core\Model\Sync\Reset;
+
+/**
+ * Class ReSetYotpoSync - Manage Yotpo Reset Sync
+ */
+class ResetYotpoSync extends Command
+{
+    const YOTPO_ENTITY              = 'entity';
+    const STORE_ID                  = 'store_id';
+    const YOTPO_ENTITY_ALL          = 'all';
+    const YOTPO_ENTITY_CUSTOMERS    = 'customer';
+    const YOTPO_ENTITY_ORDERS       = 'order';
+    const YOTPO_ENTITY_CATALOG      = 'catalog';
+
+    /**
+     * @var string[]
+     */
+    protected $yotpoEntities = ['order', 'customer', 'catalog', 'all'];
+
+    /**
+     * @var AppState
+     */
+    protected $appState;
+
+    /**
+     * @var Reset
+     */
+    protected $syncReset;
+
+    /**
+     * @var Config
+     */
+    protected $config;
+
+    /**
+     * RetryYotpoSync constructor.
+     * @param AppState $appState
+     * @param Reset $syncReset
+     * @param string|null $name
+     */
+    public function __construct(
+        Reset $syncReset,
+        AppState $appState,
+        Config $config,
+        string $name = null
+    ) {
+        $this->syncReset = $syncReset;
+        $this->appState = $appState;
+        $this->config = $config;
+        parent::__construct($name);
+    }
+
+    /**
+     * Configures the current command.
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        $options = [
+            new InputOption(
+                self::YOTPO_ENTITY,
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Entity'
+            ),
+            new InputOption(
+                self::YOTPO_ENTITY_ALL,
```

Comment:
It should be passed like below
bin/magento yotpo:resetsync --entity=all

It will then reset the sync for all the entities.

If you need to reset the sync for a particular entity , it should be passed like below.
bin/magento yotpo:resetsync --entity=order
bin/magento yotpo:resetsync --entity=customer
bin/magento yotpo:resetsync --entity=catalog

Reply:
Please check the comment [here](https://github.com/YotpoLtd/magento2-module-core/pull/104/files/85a6d866dfd576cefad350047b2c3686c2f5b208#r848294468).

Reply:
Please check the comment [here](https://github.com/YotpoLtd/magento2-module-core/pull/104/files/85a6d866dfd576cefad350047b2c3686c2f5b208#r848294468).

Reply:
Okay, we will make these changes